scripts/language_model/index.rst

```diff
@@ -183,3 +183,68 @@ The dataset used for training the models is Google's 1 billion words dataset.
 
    $ python large_word_language_model.py --gpus 0,1,2,3 --clip=10
    $ python large_word_language_model.py --gpus 4 --eval-only --batch-size=1
+
+Language Models with Transformers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reference: C. Wang, M. Li, S. Alexander. "`Language Models with Transformers`". Available soon.
```

Comment:
Given that the ref isn't available yet, could you add a paragraph of what the model does?